packages/design-system/src/components/utilities/typographyHeadings.stories.tsx

```diff
@@ -33,6 +33,17 @@ AllHeadingsOnDark.parameters = {
   baseInverse: true,
 };
 
+export const ContentClass = () => (
+  <div className="ds-content">
+    <h1>Heading level 1</h1>
```

Comment:
Could we add some paragraphs between the headers to give it more of a content block feel?

Reply:
👍 I had some in my first iteration but didn't know if it would be distracting